src/components/formStore/OneBlinkFormStoreColumnsButton.tsx

```diff
@@ -69,6 +69,9 @@ function OneBlinkFormStoreColumnsButton(
                     onChangeParameters(
                       (currentParameters) => ({
                         ...currentParameters,
+                        sorting: checked
+                          ? currentParameters.sorting
+                          : undefined,
```

Comment:
Should this be set back to the date time submitted?

Reply:
Fixed in the next one